bundles/org.openhab.binding.airgradient/src/main/java/org/openhab/binding/airgradient/internal/handler/MeasureHelper.java

```diff
@@ -67,11 +67,14 @@ public static Map<String, State> createStates(Measure measure) {
         Map<String, State> states = new HashMap<>(11);
 
         states.put(CHANNEL_ATMP, toQuantityType(measure.getTemperature(), SIUnits.CELSIUS));
+        states.put(CHANNEL_ATMP_COMPENSATED, toQuantityType(measure.atmpCompensated, SIUnits.CELSIUS));
```

Comment:
Not sure what this channel does, i don;t know how to link `compensated` with `celsius`. Will bneed documentation to verify.